templates/repo/release/list.tmpl

```diff
@@ -3,16 +3,57 @@
 	{{template "repo/header" .}}
 	<div class="ui container">
 		{{template "base/alert" .}}
-		<h2 class="ui header">
-			{{.i18n.Tr "repo.release.releases"}}
-			{{if .CanCreateRelease}}
-				<div class="ui right">
-					<a class="ui small green button" href="{{$.RepoLink}}/releases/new">
-						{{.i18n.Tr "repo.release.new_release"}}
-					</a>
-				</div>
-			{{end}}
+		<h2 class="ui compact small menu header">
+			<a class="{{if (not .PageIsTagList)}}active{{end}} item" href="{{.RepoLink}}/releases">{{.i18n.Tr "repo.release.releases"}}</a>
+			<a class="{{if .PageIsTagList}}active{{end}} item" href="{{.RepoLink}}/tags">{{.i18n.Tr "repo.release.tags"}}</a>
 		</h2>
+		{{if (and .CanCreateRelease (not .PageIsTagList))}}
+			<a class="ui right small green button" href="{{$.RepoLink}}/releases/new">
+				{{.i18n.Tr "repo.release.new_release"}}
+			</a>
+		{{end}}
+		{{if .PageIsTagList}}
+		<div class="ui divider"></div>
+		{{if gt .ReleasesNum 0}}
+		<h4 class="ui top attached header">
+			<div class="five wide column">
+				{{.i18n.Tr "repo.release.tags"}}
+			</div>
+		</h4>
+		<div class="ui attached table segment">
+			<table class="ui very basic striped fixed table single line" id="tags-table">
+				<thead></thead>
+				<tbody class="tag-list">
+					{{range $idx, $release := .Releases}}
+						<tr>
+							<td class="tag">
+								<h4>
+									<a href="{{$.RepoLink}}/src/tag/{{.TagName | EscapePound}}" rel="nofollow"><i class="tag icon"></i> {{.TagName}}</a>
+								</h4>
+								<div class="download">
+									{{if $.Permission.CanRead $.UnitTypeCode}}
+										<a href="{{$.RepoLink}}/src/commit/{{.Sha1}}" rel="nofollow"><i class="code icon"></i> {{ShortSha .Sha1}}</a>
+										<a href="{{$.RepoLink}}/archive/{{.TagName | EscapePound}}.zip" rel="nofollow">{{svg "octicon-file-zip" 16}}&nbsp;ZIP</a>
+										<a href="{{$.RepoLink}}/archive/{{.TagName | EscapePound}}.tar.gz">{{svg "octicon-file-zip" 16}}&nbsp;TAR.GZ</a>
+										{{if (and $.CanCreateRelease $release.IsTag)}}
+											<a href="{{$.RepoLink}}/releases/new?tag={{.TagName | EscapePound}}">{{svg "octicon-tag" 16}}&nbsp;{{$.i18n.Tr "repo.release.new_release"}}</a>
+											<a class="ui red delete-button" data-url="{{$.RepoLink}}/tags/delete" data-id="{{.ID}}">
+												<i class="trash icon"></i>&nbsp;{{$.i18n.Tr "repo.release.delete_tag"}}
+											</a>
+										{{end}}
+										{{if (not $release.IsTag)}}
+											<a href="{{$.RepoLink}}/releases/tag/{{.TagName | EscapePound}}">{{svg "octicon-tag" 16}}&nbsp;{{$.i18n.Tr "repo.release.detail"}}</a>
```

Comment:
Can you put a bit of margin around these links? You can add `class="mx-2"` for example to have a horizontal margin of .5rem around them.

Reply:
Also, SVG size attribute is no longer required as of recent, so you can change to format `{{svg "octicon-tag"}}`.

Also instead of that `nbsp;`, add margin directly on the svg via `{{svg "octicon-tag" 16 "mr-2"}}` (note size is required if you specifify class attributes on the svg helper).

Reply:
Did all suggestions in https://github.com/go-gitea/gitea/pull/12096#issuecomment-695789131.